all: review usage examples in docs for actuality in Typescript

If we look at https://github.com/googleapis/nodejs-spanner/blob/2a19ef1af4f6fd1b81d08afc15db76007859a0b9/src/database.ts#L3590 and try to implement it, we get back
```shell
error TS2493: Tuple type 'CommitResponse' of length '1' has no element at index '1'.

           const [resp, rerr] = await this.writeAtLeastOnce(mutations, options);
```

I've found other spots where in Typescript usage, that didn't match actual usage.

Kindly pinging @surbhigarg92 @alkatrivedi

--- a/src/database.ts
+++ b/src/database.ts
@@ -3579,7 +3579,7 @@ class Database extends common.GrpcServiceObject {
    *
    * @example
    * ```
-   * const {Spanner} = require('@google-cloud/spanner');
+   * const {Spanner, MutationSet} = require('@google-cloud/spanner');
    * const spanner = new Spanner();
    *
    * const instance = spanner.instance('my-instance');
@@ -3597,7 +3597,7 @@ class Database extends common.GrpcServiceObject {
    *  });
    *
    * try {
-   *  const [response, err] = await database.writeAtLeastOnce(mutations, {});
+   *  const [response] = await database.writeAtLeastOnce(mutations, {});
    *  console.log(response.commitTimestamp);
    * } catch(err) {
    *  console.log("Error: ", err);

--- a/test/database.ts
+++ b/test/database.ts
@@ -861,7 +861,7 @@ describe('Database', () => {
 
     it('should return CommitResponse on successful write using await', async () => {
       sinon.stub(database, 'writeAtLeastOnce').resolves([RESPONSE]);
-      const [response, err] = await database.writeAtLeastOnce(mutations, {});
+      const [response] = await database.writeAtLeastOnce(mutations, {});
       assert.deepStrictEqual(
         response.commitTimestamp,
         RESPONSE.commitTimestamp

